File: 20220111/spring/src/main/java/com/example/demo/controllers/WelcomeController.java
package com.example.demo.controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class WelcomeController {
    @GetMapping("/")
    String home() {
        return "Hello, world!";
    }
}

File: 20211018/spring/src/main/kotlin/com/example/demo/controllers/WelcomeController.kt
package com.example.demo.controllers

import com.example.demo.application.MessageGenerator
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class WelcomeController(val messageGenerator: MessageGenerator) {
    @GetMapping
    fun home(): String {
        return messageGenerator.message()
    }
}
